salt/kubernetes-minion/kubeconfig.jinja

```diff
@@ -1,13 +1,12 @@
-{% set kube_servers = [] -%}
-{% for fqdn in salt['mine.get']('roles:kube-master', 'fqdn', expr_form='grain').values() -%}
-  {% do kube_servers.append(fqdn) -%}
-{% endfor -%}
+{% set api_server = "api." + salt['pillar.get']('internal_infra_domain','')  -%}
```

Comment:
Would it make sense to use `pillar['internal_infra_domain']` and make salt directly fail? (not sure if that would make salt fail right now) -- the idea is to use some mechanism to visibly fail instead of silently set some api server that won't make sense